Makefile

```diff
@@ -113,12 +113,22 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+
+local-cross: bin/crio.cross.windows.amd64 bin/crio.cross.darwin.amd64 bin/crio.cross.linux.amd64
+
+bin/crio.cross.%:
```

Comment:
Don't we want a [`.gopathok` prereq][1] here?  I don't see why we wouldn't need it there, since we [need it for the `bin/crio` build][2].

We should also [make this target `.PHONY`][1] because we don't call `find-godeps.sh` like we [currently do for `bin/crio`][2].  I'm fine not walking in this case, because cross-platform dependency walking sounds like a pain ;).  But I do think we want these targets rebuild, instead of having `make` tell us “oh, you've already got that” after a Go dependency not listed in the `Makefile` was bumped.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1506#issuecomment-379067443
[2]: https://github.com/kubernetes-incubator/cri-o/blob/86b61d44cfe7de0f4095877d303c4d1b3aa2ec06/Makefile#L93